src/server/server.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import App from '../App';
+
+const app = express();
+
+app.get('/*', (req, res) => {
```

Comment:
Let's get this living alongside our API.

Reply:
We should also be using React to create the entire HTML document. We should be able to just return the markup returned by `ReactDOMServer.renderToString`